### PR TITLE
removed the top margin that caused a shift in the vertical centering,

### PR DESCRIPTION
**Pull request by surabi0516**
now the navbar items should be vertically centerd

### CODE REVIEW

File: frontend/src/styles/navbar/Navbar.module.css [hunk 1]
```diff
@@ -22,7 +22,6 @@
   list-style: none;
   align-items: center;
 
-  margin-top: 23px;
```
Comment (heylookitsme) [inline frontend/src/styles/navbar/Navbar.module.css:25]: uh 

image

Comment (heylookitsme) [reply frontend/src/styles/navbar/Navbar.module.css:25]: + chrome 

image

**Review by heylookitsme — CHANGES_REQUESTED** [review]
likely this is not the top margin

**Review by Stephboss9 — COMMENTED** [review]
Top margin is not reason for shift in vertical centering. Closing this PR request.

**Pull request closed without merging**